homeassistant/components/hyundai_kia_connect/coordinator.py

```diff
@@ -0,0 +1,82 @@
+"""Coordinator for Hyundai / Kia Connect integration."""
+from datetime import timedelta
+import logging
+from typing import Set
+
+from hyundai_kia_connect_api import (
+    Token,
+    VehicleManager,
+    get_implementation_by_region_brand,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_PASSWORD,
+    CONF_PIN,
+    CONF_REGION,
+    CONF_TOKEN,
+    CONF_USERNAME,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import CONF_BRAND, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class HyundaiKiaConnectDataUpdateCoordinator(DataUpdateCoordinator):
+    """Class to manage fetching data from the API."""
+
+    def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
+        """Initialize."""
+        region = config_entry.data.get(CONF_REGION)
+        brand = config_entry.data.get(CONF_BRAND)
+        username = config_entry.data.get(CONF_USERNAME)
+        password = config_entry.data.get(CONF_PASSWORD)
+        pin = config_entry.data.get(CONF_PIN)
+
+        self.platforms: Set[str] = set()
```

Comment:
```suggestion
        self.platforms: set[str] = set()
```

Reply:
hey @frenck , it is giving me this error: `homeassistant/components/hyundai_kia_connect/coordinator.py:38: error: "set" is not subscriptable, use "typing.Set" instead  [misc]`

Reply:
Add `from __future__ import annotations` to the top of your file